src/app/src/test/kotlin/org/imperial/mrc/hint/database/ProjectRepositoryTests.kt

```diff
@@ -309,8 +310,8 @@ class ProjectRepositoryTests
         assertThat(p2.sharedBy).isEqualTo(null)
         assertThat(p2.versions.count()).isEqualTo(1)
         assertThat(p2.versions[0].id).isEqualTo("v2s1")
-        assertThat(p2.versions[0].created).isEqualTo(format(ago_3h))
-        assertThat(p2.versions[0].updated).isEqualTo(format(ago_1h))
+        assertThat(roundToSec(toDateTime(p2.versions[0].created))).isEqualTo(roundToSec(ago_3h))
```

Comment:
@r-ash did a better job of this in one of his PRs, this will be a merge conflict down the line and I can replace this with his implementation, keeping it in for now so the tests pass